modules/ROOT/pages/index.adoc

```diff
@@ -94,21 +92,21 @@ a|
 
 a|
 
-* 0.7.535-4.pce30
-* 1.0.84-10.pce30
-* 0.6.993-9.pce30
-* 0.7.79
-* 0.9.64-5.pce30
-* 0.6.77-8.pce30
-* 0.6.351-1.pce30
-* 0.9.36-3.pce30
-* 0.6.9
-* 0.2.52
-* 0.8.524
-* 0.7.614-7.pce30
-* 0.8.179-10.pce30
-* 0.8.3606-9.pce30
-* 0.8.1755-5.pce30
+* v0.7.585
+* v1.0.257
```

Comment:
This is the only entry for v1, all the rest being v0?
Is this list in any particular order and, if not, how about numerical?

Reply:
The column next to it, which it supports is in Alphabetical order